CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # NMOS Common Library Changelog
 
+## 0.19.10
+- Pin pyopenssl, change authlib imports, change access token query param during websocket authentication
```

Comment:
This changelog entry appears to include things which aren't changed in the PR at present.